### PR TITLE
Cloudstack securityrule test refactor

### PR DESCRIPTION
**Pull request by fgan1**
# Description
Adding new constants to the Cloudstack Security Rule Plugin context.
# Proposed changes
- Add constants
# Additional information
Related to the "cloudstack-security-rules-test-refactor" branch in the RAS.

### CODE REVIEW

File: src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
     public static final String ERROR_CODE_KEY_JSON = "errorcode";
     public static final String ERROR_TEXT_KEY_JSON = "errortext";
+    public static final String JOB_ID_KEY_JSON = "jobid";
```
Comment (wassoares) [inline src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Remove the empty line between the constants and put them in alphabetical order.

File: src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java [hunk 2]
```diff
@@ -136,9 +135,13 @@
...
     
     public static class SecurityGroupPlugin {
+      public static final String CREATE_FIREWALL_RULE_RESPONSE = "createfirewallruleresponse";
+      public static final String DELETE_FIREWALL_RULE_RESPONSE = "deletefirewallruleresponse";
     	public static final String LIST_FIREWALL_RULES_COMMAND = "listFirewallRules";
-    	public static final String IP_ADDRESS_ID_KEY_JSON = "ipaddressid";
+      public static final String CREATE_FIREWALL_RULE_COMMAND = "createFirewallRule";
+      public static final String DELETE_FIREWALL_RULE_COMMAND = "deleteFirewallRule";
     	public static final String LIST_FIREWALL_RULES_KEY_JSON = "listfirewallrulesresponse";
+    	public static final String IP_ADDRESS_ID_KEY_JSON = "ipaddressid";
```
Comment (wassoares) [inline src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Fix the indentation of these constants in this inner class.

File: src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java [hunk 3]
```diff
@@ -136,9 +135,13 @@
     }
     
     public static class SecurityGroupPlugin {
```
Comment (wassoares) [inline src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Rename this inner class just to SecurityGroup.

Comment (fgan1) [reply src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Done

File: src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java (hunk 2 above)
Comment (fgan1) [reply src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Done

File: src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java (hunk 1 above)
Comment (fgan1) [reply src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java]: Empty line removed.
About the Alphabetical order, I will talk to you in the chat.

**Pull request merged**